pkg/rehearse/rehearse.go

```diff
@@ -638,6 +651,56 @@ func ensureImageStreamTags(ctx context.Context, client ctrlruntimeclient.Client,
 	return g.Wait()
 }
 
+func ensureISTSInQCI(ctx context.Context, client ctrlruntimeclient.Client, ists apihelper.ImageStreamTagMap,
+	mirrorOptions quayiociimagesdistributor.OCImageMirrorOptions, quayIOImageHelper quayiociimagesdistributor.OCClient, log *logrus.Entry) error {
+
+	var errs []error
+	for _, ist := range ists {
+		pairs, err := createPairs(ctx, ist, client, time.Now(), log)
+		if err == nil {
+			if err := quayIOImageHelper.ImageMirror(pairs, mirrorOptions); err != nil {
```

Comment:
Wrap the mirror into a retry like [this](https://github.com/openshift/ci-tools/blob/6359329eb9e2662b759474ba91e16eb3cd003067/pkg/controller/quay_io_ci_images_distributor/mirror.go#L111)?


We could do this later too in another PR.